app/components/course-admin/submissions-page/submission-details/autofix-request-card.hbs

```diff
@@ -1,4 +1,11 @@
 <div ...attributes>
+  {{#if (not-eq @autofixRequest.status "in_progress")}}
```

Comment:
@andy1li we can show this for in_progress too! That way one can navigate to the admin page directly and follow along